repo_tests/reach_declarations_test.py

```
@@ -0,0 +1,117 @@
+
+    Import errors are surfaced, not swallowed: a guard that cannot import is a
+    guard that is not running, and this file exists to notice exactly that class
+    of silence.
+    """
+    for module in pkgutil.iter_modules([str(_REPO_TESTS)]):
+        if module.name.startswith("_") or module.name == Path(__file__).stem:
+            continue
+        try:
+            importlib.import_module(f"repo_tests.{module.name}")
+        except Exception:  # noqa: BLE001 - collection errors belong to that module's own run
+            continue
```

Ruff (0.16.3)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)